File: go/exercise12/simple_interest.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)

	fmt.Print("Enter the principal: ")
	scanner.Scan()
	principal, _ := strconv.Atoi(scanner.Text())

	fmt.Print("Enter the rate of interest: ")
	scanner.Scan()
	rate, _ := strconv.ParseFloat(scanner.Text(), 64)

	fmt.Print("Enter the years: ")
	scanner.Scan()
	years, _ := strconv.Atoi(scanner.Text())

	interest := (float64(principal) * (1 + float64(years)*(rate/100)))

	fmt.Printf("After %v years at %.1f%%, the investment will be worth $%v.\n", years, rate, interest)

}

File: javascript/exercise12/simple-interest.js
import { input } from "pynput"

const principal = Number(await input("Enter the principal: "))
const rate = Number(await input("Enter the rate of interest: "))
const years = Number(await input("Enter the years: "))

const interest = (principal.toFixed(2) * (1 + (years * (rate / 100))))

console.log(`After ${years} years at ${rate}%, the investment will be worth $${interest}.`)
